### ISSUE TITLE
Tests are broken when running tox

### ISSUE THREAD

**Issue by medecau** [cobrateam/splinter#776]
I have tried to run the tests using tox. Tried on py36, py37, and py38, all produce the following error.

`
___ ERROR collecting samples/test_google_search.py ___
samples/test_google_search.py:35: in <module>
    unittest.main()
../../.pyenv/versions/3.6.10/lib/python3.6/unittest/main.py:95: in __init__
    self.runTests()
../../.pyenv/versions/3.6.10/lib/python3.6/unittest/main.py:258: in runTests
    sys.exit(not self.result.wasSuccessful())
E   SystemExit: False
`

**Comment by jsfehler** [comment]
Most likely, you ran tox without any arguments and picked up the samples directory.

**Comment by medecau** [comment]
I had to run `tox -- tests` in order to get pytest to pick only the tests folder.

The tests are now running up to 48% where they hang-up and do not seem to progress.
Last HTTP request being:
`127.0.0.1 - - [23/May/2020 20:20:20] "GET /alert HTTP/1.1" 200 -`

**Comment by jsfehler** [comment]
What browser/driver/OS versions are you using?

**Comment by medecau** [comment]
macOS Mojave 10.14.6
geckodriver installed from brew
Firefox 76.0.1

```
$uname -mpsr
Darwin 18.7.0 x86_64 i386
$brew info geckodriver             
geckodriver: stable 0.26.0 (bottled), HEAD
WebDriver <-> Marionette proxy
[...]
```

**Comment by jsfehler** [comment]
Looks like the tests are failing on the latest version of Firefox, CI is hard-coded to Firefox 68 so it went undetected.

**Comment by medecau** [comment]
Thank you for looking into this

**Comment by jsfehler** [comment]
Firefox failures fixed by #782
